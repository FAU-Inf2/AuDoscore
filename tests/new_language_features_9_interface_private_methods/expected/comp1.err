ERROR - Method public static int IToTest.toTest_static()[IToTest] does not match or does not exist in student code
Exception in thread "main" java.lang.Error
	at tools.ic.InterfaceComparator.main(InterfaceComparator.java:335)
make[1]: *** [Makefile:119: run-comparer] Fehler 1
make: *** [Makefile:73: compile-stage1] Fehler 2
