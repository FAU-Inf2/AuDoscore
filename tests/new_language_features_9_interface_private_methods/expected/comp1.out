rm -f *.class */*.class
javac  -sourcepath interfaces student/*.java
javac -Xprefer:source  -sourcepath interfaces:cleanroom -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar cleanroom/*.java
javac -Xprefer:source  -sourcepath interfaces:junit:student -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar junit/PublicTest.java
java -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar:junit:interfaces:student CheckAnnotation PublicTest
java -cp lib/junitpoints.jar:lib/junit.jar:lib/hamcrest-core.jar:lib/json-simple-1.1.1.jar tools.ForbiddenUseSearcher PublicTest > forbidden.out
if [ -s forbidden.out ]; then \
	cat forbidden.out 1>&2 ; \
	exit 1 ; \
fi
make run-comparer
make[1]: Verzeichnis „/audoscore/tests/new_language_features_9_interface_private_methods/test.162628“ wird betreten
java -cp lib/junitpoints.jar tools.ic.InterfaceComparator PublicTest
make[1]: Verzeichnis „/audoscore/tests/new_language_features_9_interface_private_methods/test.162628“ wird verlassen
